seacrowd/sea_datasets/cc100/cc100.py

```diff
@@ -135,9 +135,17 @@
 _LICENSE = "MIT"
 
 _LANGUAGES_MAP = {
-    "ind": "id",
-    "jav": "jv",
-    "sun": "su",
+    "ind": "id", # Indonesia
+    "jav": "jv", # Java
+    "sun": "su", # Sundanese
+    "mya": "my", # Burmese
+    "mya_zaw": "my_zaw", # Burmese (Zawgyi)
+    "lao": "lo", # Lao
+    "khm": "km", #Central Khmer, Khmer
+    "tgl": "tl", # Tagalog
+    "vie": "vi", # Vietnamese
+    "tha": "th", # Thai
+    "zlm": "ms", # Malay 
```

Comment:
We also need to update the `_LANGUAGES` constant in Line 46 to include these new languages.

Reply:
Ah, forgot to do that! Updated 👍